src/confdb/confdb.c

```diff
@@ -1029,6 +1029,18 @@ static int confdb_get_domain_internal(struct confdb_ctx *cdb,
         domain->pwfield = "x";
     }
 
+    if (domain->provider != NULL && strcmp(domain->provider, "proxy") == 0) {
```

Comment:
`is_files_provider()` uses `strcasecmp()` for similar check

Reply:
Good idea. I switched it to strcasecmp.